go.mod

```diff
@@ -50,8 +50,8 @@ require (
 	github.com/filecoin-project/specs-actors/v4 v4.0.1
 	github.com/filecoin-project/specs-actors/v5 v5.0.4
 	github.com/filecoin-project/specs-actors/v6 v6.0.1
-	github.com/filecoin-project/specs-actors/v7 v7.0.0-20211118013026-3dce48197cec
-	github.com/filecoin-project/specs-storage v0.1.1-0.20211213202648-f14267c929ff
+	github.com/filecoin-project/specs-actors/v7 v7.0.0-20211230214648-aeae366b083a
+	github.com/filecoin-project/specs-storage v0.1.1-0.20211228030229-6d460d25a0c9
```

Comment:
Use tagged versions before landing in master.

Reply:
Thanks, https://github.com/filecoin-project/specs-storage/pull/20